manifests/wildfly/1.0.0/manifests/wildflyservers.wildfly.org.crd.yaml

```diff
@@ -1064,18 +1064,25 @@ spec:
                     podIP:
                       type: string
                     state:
-                      description: Represent the state of the Pod, it is used especially
-                        during scale down.
+                      description: Represents the state of the Pod, it is used during scale down.
```

Comment:
This file should not be modified, this is the CDR for 1.0.0, a version already released

Reply:
Done. Thanks